pom.xml

```diff
@@ -55,7 +55,7 @@
 			we need to compile against version 23. All such classes are in the org.acra.jraf package.
 		-->
         	<android.version>23</android.version>
-		<support.version>23.2.1</support.version>
+		<support.version>23.3.0</support.version>

```

Comment:
Don't be so hasty in bumping the versions of the support libraries.
We should only do that if ACRA needs it.
Otherwise we are shoving new versions of the support libraries down people's throats.
That should be a decision for them not us.